.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn commitlint --edit 
```

Comment:
This file was created automatically with the yarn husky add .husky/pre-commit "yarn commitlint --edit" command.

What is tells is that Husky should execute `yarn commitlint --edit` on top of the commit message, before creating the commit itself. Also, we tell `commitlint` that it should open the editor, so the user can further alter the message if needed (that's the default behavior of Git, so we maintain it here).